website/src/parsers/markdown/transformers/remark/index.js

```diff
@@ -1,5 +1,20 @@
 import compileModule from '../../../utils/compileModule';
 import pkg from 'remark/package.json';
+import unistUtilIs from 'unist-util-is';
+import unistUtilVisit from 'unist-util-visit';
+import unistUtilVisitParents from 'unist-util-visit-parents';
```

Comment:
I'd prefer if those are also loaded dynamically, but if they are not too large then it's OK for now. You could create the `sandboxRequire` function in the `loadTransformer` method to make it available inside `transform`.

Reply:
addressed in 0e12f3c